mongoexport/mongoexport.go

```diff
@@ -33,6 +33,8 @@ const (
 	CSV                            = "csv"
 	JSON                           = "json"
 	watchProgressorUpdateFrequency = 8000
+	canonical                      = "canonical"
```

Comment:
[nit] the comment above says these constants are the different output types, which isn't exactly true. I'd say either update the comment or break out the constants defined here into logical categories (e.g. output formats, extended json type, etc)

Reply:
added a jsonFormat type and split up the constants in new commit.